scanner/main.c

```diff
@@ -141,6 +141,50 @@ static void write_to_LED(uint8_t conn_handle, uint32_t value, uint8_t char_index
     NRF_LOG_INFO("Wrote to the characteristic.");
 }
 
+static void turn_on_RGB_LED(uint32_t color_code)
+{
+    switch(color_code)
+    {
+        case RED:
+            bsp_board_led_on(1);    bsp_board_led_off(2);

```

Comment:
The formatting of the LED on/off commands is kind of confusing. I would suggest to keep each of them in a separate row and order them as 1..2..3 in each case.